### DISCUSSION TITLE
V2 Search only returning tags

### DISCUSSION THREAD

**Discussion by natecroft** [statamic/cms#5203] [Troubleshooting]
I have a V2 site that wanted to add search. 

Search tag worked great, then for reason I am unaware of, started to only return tags. No pages or entries, only tags. 
I've cleared the cache and no luck. Any ideas?

Even running the simplest of requested data {{ title }} in the results template does this.

**Comment by duncanmcclean** [comment]
Apologies for the late reply but would you be able to share your code? Maybe we'll be able to pick up what's happening.

**Reply by natecroft** [reply]
Duncan, 

Absolutely, here is the code. Note: This worked for a while then mysterious stopped. 

`
<div class="row single-place-info">
	

	
		<div class="col-md-8 main-place-info search-results-page">
			
			<h1 class="places-title">Search</h1>
						<section class="regular">
				
				<h2>Search Results</h2>
				
				<article class="content">
					
					{{ search:results }}
					
						{{ if no_results }}
							<p>We didn't find results for that. Want to try again?</p>
							<form class="onpage-search-form"action="/search" method="GET">
							  <input type="text" name="q" /><br /><br />
							  <button class="btn-blue">Search</button>
							</form>
						
						{{ /if }}
						
							 
							<div class="block">
							
							{{#
			
								{{ if main_image  }}
									<a href="{{ url }}">
										<img src="{{ glide src='{ main_image }' square='140' fit='crop' }}" width="140" height="140" class="small-image search-preview-img">
									</a>
								{{ elseif content_main_image  }}
									<img src="{{ glide src='{ content_main_image }' square='140' fit='crop' }}" width="140" height="140" class="small-image">
								{{ /if }}
								
							#}}
								
								{{ if title }}
								<a href="{{ url }}">
									<h3>{{ title }}</h3>
								</a>
								{{ /if }}
								
							{{#	
								<p>
									{{ if post_content }}
										{{ post_content strip_tags="img|a|p" safe_truncate="180|..." }}
									{{ /if }}
									
									{{ if details }}
										{{ details strip_tags="img|a|p" safe_truncate="180|..." }}
									{{ /if }}
									
									{{ if event_description }}
										{{ event_description strip_tags="img|a|p" safe_truncate="180|..." }}
									{{ /if }}
								</p>
							#}}
			
								
							</div> 
							
						
					{{ /search:results }}
					
				</article>
			</section>

						
			
			
		</div>

	
	
	
	
	
	
	<div class="col-md-3 offset-md-1 side-bar">
		
			<ul>
				
			
				
					<h3>Search</h3>
					<form class="sidebar-search-form"action="/search" method="GET">
					  <input type="text" name="q" />
					  <button class="btn-blue">Search</button>
					</form>
				
			
				
				<h3>See More Articles</h3>
					{{ collection:blog limit="7" sort="random" title:isnt="{ title }" }}
					<li><a href="{{ url }}">{{ title }}</a></li>
					{{ /collection:blog }}
					
		</ul>
		
		
	</div>
	
</div>











`

**Reply by duncanmcclean** [reply]
That looks fine to me at a glance. 

Can you maybe try removing the Antlers comments in the template? If that's still not working, can you share a screenshot of what you're seeing?

**Reply by natecroft** [reply]
Duncan, 

Thanks for having a look. I have removed the Antlers comments and the same thing. Nothing happens. At one time this worked beautifully. Attaching a screenshot
![Screen Shot 2022-02-22 at 10 42 51 AM](https://user-images.githubusercontent.com/2131208/155167124-b68d1f49-dc7b-4b45-8bbd-f369202e9ff4.png)
.

**Reply by duncanmcclean** [reply]
Oh, I was thinking it was returning Antlers tags in a weird way or something but now I get it, it's literally returning tags (a taxonomy), am I right?

What does your search config look like? `site/settings/search.yaml`

**Reply by natecroft** [reply]
It looks like this:

driver: local
auto_index: true
default_index: default
searchable:
  - title
  - content
